dns.go

```diff
@@ -7,25 +7,36 @@ import (
 	ma "github.com/multiformats/go-multiaddr"
 )
 
+// You **MUST** register your multicodecs with
+// https://github.com/multiformats/multicodec before adding them here.
+//
+// TODO: Use a single source of truth for all multicodecs instead of
```

Comment:
The table is the source of truth. I think it was decided it was ok to have each package define their Codes themselves rather than having a centralized Go module, so I would not add a TODO about doing things differently.